packages/itwinui-react/src/core/utils/components/InputWithIcon.tsx

```diff
@@ -0,0 +1,8 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { polymorphic } from '../index.js';
+
+export const InputWithIcon = polymorphic.div('iui-input-with-icon');
```

Comment:
Should this be marked as `@private`?

Reply:
should this comment be marked as "nit"? 😅

Reply:
Since I don't see any changeset saying that `InputWithIcon` is publically exposed, I think this was meant to be a private component. If so, then I think `@private` is important.

Reply:
It's not important because it is not exposed in the barrel file. The user cannot import it.

The whole `@private` thing used to only matter in previous major versions when we were exposing internal sub-paths